src/translations/extras/extras.xliff

```diff
@@ -162,7 +162,7 @@
       <trans-unit id="servers.br" xml:space="preserve">
         <source>Brazil</source>
       </trans-unit>
-      <trans-unit id="servers.SaoPaulo" xml:space="preserve">
+      <trans-unit id="servers.SãoPaulo" xml:space="preserve">
```

Comment:
Is this strictly necessary? This file is translated, this string will show up as new.

Reply:
Argh! Great catch... I just wanted to change this in the test extras.xliff files 🙄 
```suggestion
      <trans-unit id="servers.SaoPaulo" xml:space="preserve">
```